packages/reports/addon/components/report-view.js

```diff
@@ -112,6 +112,16 @@ export default Ember.Component.extend({
     Ember.run.scheduleOnce('afterRender', this, 'resizeVisualization');
   }),
 
+  /**
+   * Recomputes visualization so it stays in sync after updating
+   * metrics/dimensions when editing pane is open.
+   *
+   * @method hasReportRunDidChange
```

Comment:
jsdoc style for method:
https://github.com/yahoo/navi/blob/master/packages/reports/addon/components/dimension-bulk-import.js#L135